daemon/config.go

```diff
@@ -82,6 +82,7 @@ func (h *patchConfig) Handle(params PatchConfigParams) middleware.Responder {
 	changes += option.Config.Opts.ApplyValidated(om, changedOption, d)
 
 	log.WithField("count", changes).Debug("Applied changes to daemon's configuration")
+	option.Config.ConfigPatchMutex.Unlock()
```

Comment:
Consider having this code in a different function that does the Lock/Unlock to avoid forgetting to do an `Unlock()` when we will add a new return.

Reply:
Will create a good first issue to refactor this